ovos_core/intent_services/fallback_service.py

```diff
@@ -168,8 +177,9 @@ def _fallback_range(self, utterances, lang, message, fb_range):
 
         sess = SessionManager.get(message)
         # new style bus api
+        available_skills = self._collect_fallback_skills(message, fb_range)
         fallbacks = [(k, v) for k, v in self.registered_fallbacks.items()
-                     if k in self._collect_fallback_skills(message, fb_range)]
```

Comment:
NOTE for reviewers: this would repeat the ping/pong bus message pattern once per skill, it can't be used inside the iterator